hyp3_testing/templates/opera_rtc_s1_golden.json.j2

```diff
@@ -0,0 +1,29 @@
+[
+  {
+    "job_type": "OPERA_RTC_S1",
+    "name": "{{ name }}",
+    "job_parameters": {
+      "granules": [
+        "S1_366240_IW1_20220321T103707_VV_8575-BURST"
+      ]
+    }
+  },
+  {
+    "job_type": "OPERA_RTC_S1",
+    "name": "{{ name }}",
+    "job_parameters": {
+      "granules": [
+        "S1_073251_IW2_20220111T020806_VV_9D98-BURST"
+      ]
+    }
+  },
+  {
+    "job_type": "OPERA_RTC_S1",
+    "name": "{{ name }}",
+    "job_parameters": {
+      "granules": [
+        "S1_209152_IW2_20220103T101538_HH_BD12-BURST"
```

Comment:
Could add a Jinja comment like `{# comment #}` to each of these test cases explaining its significance / why it was chosen, or put that in the README if a longer explanation is required.